.github/workflows/verifyHybridApp.yml

```diff
@@ -0,0 +1,141 @@
+name: Verify HybridApp build
+
+on:
+  workflow_call:
```

Comment:
Did you mean to include `workflow_dispatch` trigger? `workflow_call` is for reusable, callable workflows. But we don't have any such usage of this new workflow.

Reply:
I will remove 👍